tests/Backend/Feature/api/v1/Room/FileTest.php

```
@@ -389,42 +589,300 @@ public function test_download_files_download_with_access_code()
-     * Test get download url of file that is not with participants of a room
+     * Test get download url of file that is not downloadable with participants of a room
      */
     public function test_download_files_download_disabled()
     {
-        $this->actingAs($this->room->owner)->postJson(route('api.v1.rooms.files.get', ['room' => $this->room]), ['file' => $this->file_valid])
+        $this->actingAs($this->room->owner)->postJson(route('api.v1.rooms.files.add', ['room' => $this->room]), ['file' => $this->file_valid])
             ->assertSuccessful();
         $this->room->allow_guests = true;
         $this->room->save();
 
         $room_file = $this->room->files()->where('filename', $this->file_valid->name)->first();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$room_file'. (undefined)

(UnusedLocalVariable)